main.tf

```diff
@@ -18,6 +18,7 @@ resource "aws_efs_file_system" "this" {
 
     content {
       transition_to_ia                    = try(lifecycle_policy.value.transition_to_ia, null)
+      transition_to_archive               = try(lifecycle_policy.value.transition_to_archive, null)
```

Comment:
hmm, why are we not seeing this same change being picked up in the wrappers?

Reply:
Because `transition_to_archive` is just a key inside of `var.lifecycle_policy`

Reply:
ah, good point